### ISSUE TITLE
Localization of known folders names

### ISSUE THREAD

**Issue by thomascube** [thomascube/roundcube-trac-import#55]
**Reported by marcinkk on 23 May 2007 19:19 UTC**

Hello,

  I've configured, for example:

```
$rcmail_config[= 'saved-drafts';
...

$rcmail_config['default_imap_folders']('drafts_mbox']) = array('INBOX', 'saved-drafts', 'sent-mail', 'spam-mail', 'mail-trash');
```

  And after this configuration known folders like drafts, sent, junk and trash are not localized. I've added to labels.inc:

```
$labels[= 'Szkice';
...
```

  But I think it should be done better. If I can set ```$rcmail_config['drafts_mbox']('saved-drafts'])```, then the localized name of Drafts folder should be set to folder from ```$config['drafts_mbox']```. I don't know how to do it :(

Migrated-From: http://trac.roundcube.net/ticket/1484390

**Comment by thomascube** [comment]
**Comment by thomascube on 25 May 2007 22:05 UTC**
This is currently hard coded in program/steps/mail/func.inc, line 112
```
// for these mailboxes we have localized labels
$special_mailboxes = array('inbox', 'sent', 'drafts', 'trash', 'junk');
```

**Comment by thomascube** [comment]
**Modified by thomascube on 25 May 2007 22:10 UTC**

**Comment by thomascube** [comment]
**Comment by thomascube on 12 Feb 2008 18:48 UTC**
Seems to be a dupe of #1484707 which was fixed in [cb3badb7]
